packages/grafana-schema/src/schema/dashboard/v2alpha0/types.gen.ts

```diff
@@ -871,6 +871,43 @@ export const defaultResponsiveGridLayoutItemSpec = (): ResponsiveGridLayoutItemS
 	element: defaultElementReference(),
 });
 
+export interface TabsLayoutKind {
+	kind: "TabsLayout";
+	spec: TabsLayoutSpec;
+}
+
+export const defaultTabsLayoutKind = (): TabsLayoutKind => ({
+	kind: "TabsLayout",
+	spec: defaultTabsLayoutSpec(),
+});
+
+export interface TabsLayoutSpec {
+	tabs: TabItemKind[];
+}
+
+export const defaultTabsLayoutSpec = (): TabsLayoutSpec => ({
+	tabs: [],
+});
+
+export interface TabItemKind {
+	kind: "TabItem";
+	spec: TabItemSpec;
+}
```

Comment:
Just to keep the same nomenclature, we could name it as
```suggestion
export interface TabsLayoutItemKind {
	kind: "TabsLayoutItem";
	spec: TabsLayoutItemSpec;
}
```

wdyt?

Reply:
Yes. Good point

Reply:
I'd like to change this to TabLayoutTabKind instead of item. We associate item with the grid items that correspond to the panel containers/panel chrome.

Reply:
Could I suggest `TabsLayoutTabKind`? 🙈